scanner/services/matcher.go

```diff
@@ -193,7 +194,11 @@ func (s *matcherService) GetSBOM(ctx context.Context, req *v4.GetSBOMRequest) (*
 		return nil, err
 	}
 
-	sbom, err := s.matcher.GetSBOM(ctx, ir, req.GetName(), req.GetId())
+	sbom, err := s.matcher.GetSBOM(ctx, ir, &sbom.Options{
+		Name:      req.GetId(),
+		Namespace: req.GetUri(),
+		Comment:   fmt.Sprintf("Tech Preview - generated for '%s'", req.GetName()),
```

Comment:
do you prefer `'` or `"`? As in, keep as-is or use `%q` instead?

Reply:
When the double quote is encoded in JSON it will look like 

```
 "comment": "Tech Preview - generated for \"docker.io/library/nginx:latest\"",
```

Thought it was cleaner with the single quote

```
 "comment": "Tech Preview - generated for 'docker.io/library/nginx:latest'",
```

Am OK putting whatever here, or removing the quotes entirely.

Reply:
eh it's tech preview. good for now. we can always change later (probably will since this explicitly says "Tech Preview" anyway)